src/internal/common/str/str.go

```diff
@@ -90,3 +92,25 @@ func SliceToMap(ss []string) map[string]struct{} {
 
 	return m
 }
+
+func GenerateHash(input []byte, hashLength int) string {
+	h := md5.New()
+	h.Write([]byte(input))
+	hexstr := hex.EncodeToString(h.Sum(nil))
+
+	if hashLength > 0 {
+		return hexstr[0:hashLength]
+	}
+
+	return hexstr
+}
+
+func Contains(slice []string, item string) bool {
```

Comment:
We can use [`slices.Contains`](https://pkg.go.dev/golang.org/x/exp/slices#Contains) instead.